rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -321,6 +333,18 @@ def verify_event_record(event_type, bucket, event_record_path, ceph_version):
             else:
                 raise EventRecordDataError("awsRegion not in event record")
 
+            # verify object version in case versioning is enabled
+            if versioning:
+                # get object version id using radosgw-admin cli
+                object_key = event_record_json["Records"][0]["s3"]["object"]["key"]
```

Comment:
Do we need this value assignment..since we are not using object_key anywhere in the script?

Reply:
@ckulal , yes, we are fetching the versionID from the event_record into object_key

Reply:
In line no 340 versionId is fetched from event record and assigned to variable: event_version_id
and that variable is used for further condition check.

but in line no 339: key is fetched from event record and assigned to variable: object_key
which is unused